# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-print10/freetype.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fixed header inclusion for freetype2 as of 2.0.7.

  - Rene Rebe <reneÂ@exactcode.de>

--- gnome-print-0.35/libgnomeprint/gnome-font-private.h	2001-12-05 23:07:58.000000000 +0100
+++ gnome-print-0.35-fixed/libgnomeprint/gnome-font-private.h	2004-12-22 11:56:41.417915064 +0100
@@ -22,6 +22,7 @@
 
 BEGIN_GNOME_DECLS
 
+#include <ft2build.h>
 #include <freetype/freetype.h>
 #include <libgnomeprint/gp-character-block.h>
 #include <libgnomeprint/gp-fontmap.h>
--- gnome-print-0.35/libgnomeprint/parseTT.h.orig	2004-12-22 11:24:04.747091056 +0000
+++ gnome-print-0.35/libgnomeprint/parseTT.h	2004-12-22 11:24:17.760112776 +0000
@@ -22,6 +22,7 @@
 BEGIN_GNOME_DECLS
 
 #include <glib.h>
+#include <ft2build.h>
 #include <freetype/freetype.h>
 
 /*
--- gnome-print-0.35//installer/gf-pfb.c.orig	2004-12-22 11:33:48.576335520 +0000
+++ gnome-print-0.35-fixed/installer/gf-pfb.c	2004-12-22 11:34:01.435380648 +0000
@@ -12,6 +12,7 @@
 #include <fcntl.h>
 #include <string.h>
 #include <ctype.h>
+#include <ft2build.h>
 #include <freetype/freetype.h>
 #include "gf-pfb.h"
 
--- gnome-print-0.35//installer/gf-ttf.c.orig	2004-12-22 11:34:16.269125576 +0000
+++ gnome-print-0.35-fixed/installer/gf-ttf.c	2004-12-22 11:34:28.814218432 +0000
@@ -32,6 +32,7 @@
 #include <fcntl.h>
 #include <string.h>
 #include <ctype.h>
+#include <ft2build.h>
 #include <freetype/freetype.h>
 
 #include "gf-ttf.h"
--- gnome-print-0.35/installer/gnome-font-install.c.orig	2004-12-22 11:32:48.285501120 +0000
+++ gnome-print-0.35-fixed/installer/gnome-font-install.c	2004-12-22 11:33:22.743262744 +0000
@@ -26,6 +26,7 @@
 #include <parser.h>
 #include <xmlmemory.h>
 /* End of ugly thing */
+#include <ft2build.h>
 #include <freetype/freetype.h>
 #include <libgnome/gnome-defs.h>
 #include <libgnome/gnome-util.h>
